system/Languages/LanguageBundle.properties

```diff
@@ -22,8 +22,8 @@ in_alignLabel = Alignment
 in_sourceLabel = Source
 in_orientation = Click to chang orientation of tables
 in_infoScrollTip = Any requirements you don't meet are in italics
-in_qualifyMess = You do not qualify for 
```

Comment:
As bad as the choice may be to have a space here - are you sure the code that uses this doesn't expect it?  How is this processed in practice?